istioctl/pkg/waypoint/waypoint.go

```diff
@@ -61,6 +62,8 @@ const (
 	noTraffic       = "none"
 )
 
+var validAddressTypes = sets.New(serviceTraffic, workloadTraffic, allTraffic, noTraffic)
```

Comment:
Can we also name this to validTrafficTypes? I was including these refactors in my current [PR ](https://github.com/istio/istio/pull/49970/files) but since you're here, might as well update that now.